lib/src/model/in_app_purchase.dart

```diff
@@ -0,0 +1,178 @@
+import 'package:appstore_connect/src/model/model.dart';
+
+class Territory extends Model {
+  static const String type = 'territories';
+  static final Territory USA = Territory('USA', {'currency': 'USD'});
+
+  final String id;
+  final String currency;
+
+  Territory(String id, Map<String, dynamic> attributes)
+      : id = id,
+        currency = attributes['currency'],
+        super(type, id);
+
+}
```

Comment:
[Anmerkung] Vielleicht könnten ein paar Models hieraus auch in eigenen Dateien leben.

Reply:
Ich hab Territory in ein eigenes File verschoben. Das macht Sinn. Die restlichen Models sind Teil des iap "Namespaces" und ich würde sie in dem File erwarten. Oder hab ich was übersehen?

UploadOperation & RequestHeader sind eher Hilfsklassen und erben nicht von Model. Hätte sie dementsprechend nicht in ein anderes File verschoben.